### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by omahs**
docs: fix typos

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct `docs:` prefix in the PR title
- [x] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/interchain-security/blob/main/CONTRIBUTING.md#pr-targeting))
- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary 
- [x] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] Confirmed the correct `docs:` prefix in the PR title
- [ ] Confirmed all author checklist items have been addressed 
- [ ] Confirmed that this PR only changes documentation
- [ ] Reviewed content for consistency
- [ ] Reviewed content for spelling and grammar
- [ ] Tested instructions (if applicable)
- [ ] Checked that the documentation website can be built and deployed successfully (run `make build-docs`)

### CODE REVIEW

**Review by MSalopek — APPROVED** [review]
LGTM

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Please go the the `Preview` tab and select the appropriate sub-template:
+Please go to the `Preview` tab and select the appropriate sub-template:
 
 * [Production code](?expand=1&template=production.md) - for types `fix`, `feat`, and `refactor`.
 * [Docs](?expand=1&template=docs.md) - for documentation changes.
-* [Others](?expand=1&template=others.md) - for changes that do not affect production code.
\ No newline at end of file
+* [Others](?expand=1&template=others.md) - for changes that do not affect production code.
```
Comment (p-offtermatt) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: Why does this line show as modified? I can't spot any difference, and a diffchecker also claims they are the same. Is there something subtle?

**Review by p-offtermatt — COMMENTED** [review]
LGTM, but one concern about a change

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (omahs) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: Hey, it's a bug on the Github web editor. It sometimes does this to the last line of markdown files.

Comment (p-offtermatt) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: Cool, thanks for clarifying.

**Review by p-offtermatt — APPROVED** [review]

**Pull request merged**